back-end/Controllers/userController.js

```diff
@@ -0,0 +1,137 @@
+const userData = require("../Schema/userData");
+const userDetails = require("../Schema/Details");
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+const SALT = 10
```

Comment:
Place all the Variables which are related to functionality inside the respective controllers or middlewares